### DISCUSSION TITLE
Add Fast and slow judgement

### DISCUSSION THREAD

**Discussion by angealoi** [ppy/osu#25962] [Ideas]
Why need fast and slow judgement ?

1. UR bar is very hard to read ingame 

2. More easy to know my tapping is too slow or fast 

How it works? :

![222](https://github.com/ppy/osu/assets/154448118/5d9fabf5-692d-4654-9b3d-f8d637637533)

It just marking UR Bar as a judgement , fast? too early ? fast judgement , too late ? slow judgement

Mania have 320 judgement but std mod no 320 judgement so should set arbitrarily fast/slow 300 judgement ms range

So Please decide this ppy.

Osu Mania example:

![Honeycam 2023-12-21 00-14-00](https://github.com/ppy/osu/assets/154448118/ac41ae34-add9-4b2e-838f-70874d35d09a)

Osu std example ( My thoughts ):

Add more judgement image like this

slow 300 200 100 50 miss .png , 300 200 100 50 miss .png , fast 300 200 100 50 miss .png

![explorer_oSyh1cLhVf](https://github.com/ppy/osu/assets/154448118/dcb6489a-5126-42c7-8501-5c3a3a879d61)

**Comment by peppy** [comment]
https://github.com/ppy/osu/discussions/23172 and probably 10 more discussions already posted.

We have a WIP design for this, it will eventually come.
